src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -382,13 +446,20 @@ def create_pod_object(
         )
 
     def create_bitcoind_service(self, tank) -> client.V1Service:
-        service_name = f"bitcoind-service-{tank.index}"
+        service_name = f"warnet-{POD_PREFIX}-{tank.index:06d}"
```

Comment:
Being as the lightning service is called "lightning-service-01" shouldn't we keep the bitcoind service named "bitcoind-service-01" too?

Reply:
Yeah I was actually thinking this too...